modules/gapi/include/opencv2/gapi/gcommon.hpp

```diff
@@ -21,6 +21,8 @@
 
 namespace cv {
 
+class GMat; // FIXME: forward declaration for GOpaqueTraits
```

Comment:
@dmatveev please, take a look at the last 2 commits

Reply:
Done, thanks!